diff -u -r original/davix.spec modified/davix.spec
--- original/davix.spec	2020-05-05 17:19:37.000000000 +0200
+++ modified/davix.spec	2020-06-03 14:27:15.327981177 +0200
@@ -24,11 +24,6 @@
 # davix-copy dependencies
 BuildRequires:                  gsoap-devel
 
-# unit tests and abi check
-%if %{?fedora}%{!?fedora:0} >= 10 || %{?rhel}%{!?rhel:0} >= 6
-BuildRequires:                  abi-compliance-checker
-%endif
-
 Requires:                       %{name}-libs%{?_isa} = %{version}-%{release}
 
 
@@ -92,9 +87,6 @@
 make %{?_smp_mflags}
 make doc
 
-%check
-ctest -V -T Test
-
 
 %install
 rm -rf %{buildroot}

