diff -u -r original/xrootd.spec modified/xrootd.spec
--- original/xrootd.spec	2020-07-07 15:25:31.000000000 +0200
+++ modified/xrootd.spec	2020-07-16 11:11:53.912824270 +0200
@@ -98,11 +98,6 @@
 BuildRequires: json-c-devel
 
 BuildRequires: python2-devel
-%if %{?fedora}%{!?fedora:0} >= 13
-BuildRequires: python3-devel
-%else
-BuildRequires: python%{python3_pkgversion}-devel
-%endif
 
 BuildRequires: openssl-devel
 
@@ -319,21 +314,6 @@
 %description -n python2-xrootd
 Python 2 bindings for XRootD
 
-%if %{?_with_python3:1}%{!?_with_python3:0}
-#-------------------------------------------------------------------------------
-# python3
-#-------------------------------------------------------------------------------
-%package -n python3-%{name}
-Summary:       Python 3 bindings for XRootD
-Group:         Development/Libraries
-%if %{?fedora}%{!?fedora:0} >= 13
-%{?python_provide:%python_provide python3-%{name}}
-%endif
-Requires:      %{name}-client-libs%{?_isa} = %{epoch}:%{version}-%{release}
-
-%description -n python3-xrootd
-Python 3 bindings for XRootD
-%endif
 
 #-------------------------------------------------------------------------------
 # doc
@@ -464,12 +444,6 @@
 
 doxygen Doxyfile
 
-%if %{?_with_python3:1}%{!?_with_python3:0}
-# build python3 bindings
-pushd build/bindings/python
-%py3_build
-popd
-%endif
 
 #-------------------------------------------------------------------------------
 # Installation
@@ -562,12 +536,6 @@
 install -m 644 -p packaging/common/xrootd.pp \
     %{buildroot}%{_datadir}/selinux/packages/%{name}/%{name}.pp
 
-%if %{?_with_python3:1}%{!?_with_python3:0}
-# install python3 bindings
-pushd build/bindings/python
-%py3_install
-popd
-%endif
 
 %clean
 rm -rf $RPM_BUILD_ROOT
@@ -861,12 +829,6 @@
 %doc %{_mandir}/man1/libXrdVoms.1.gz
 %doc %{_mandir}/man1/libXrdSecgsiVOMS.1.gz
 
-%if %{?_with_python3:1}%{!?_with_python3:0}
-%files -n python3-%{name}
-%defattr(-,root,root,-)
-%{python3_sitearch}/*
-%endif
-
 %files doc
 %defattr(-,root,root,-)
 %doc %{_docdir}/%{name}-%{version}
